db/seeds.rb

```diff
@@ -6,7 +6,11 @@
 FactoryGirl.create(:skill, name: "skill2", featured: true)
 FactoryGirl.create(:skill, name: "skill2", featured: false)
 
+puts " Successfully added skills from seed data. ".center(70, "~*~')
+
 FactoryGirl.create(:group, name: "1407")
 FactoryGirl.create(:group, name: "1409")
 FactoryGirl.create(:group, name: "1410")
 FactoryGirl.create(:group, name: "1412")
+
+puts " Successfully added groups from seed data. ".center(70, "~*~")
```

Comment:
unexpected token tCONSTANT<br>unexpected token tSTRING<br>unterminated string meets end of file